pynamodb/connection/dax.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+from amazondax import AmazonDaxClient
```

Comment:
Perhaps by moving the import into the `__init__` we could make this dependency optional.